IMPORTING MODULE
	>>> module = __import__('7-base_geometry')

IMPORTING FUNCTION
	>>> BaseGeometry = __import__('7-base_geometry').BaseGeometry

CHECKING CREATE INSTANCE
	>>> bg = BaseGeometry()

CHECKING DOCSTRING MODULE
	>>> len(module.__doc__) > 1
	True

CHECKING DOCSTRING FUNCTION
	>>> len(BaseGeometry.__doc__) > 1
	True

CHECKING TWO STRING
	>>> bg.integer_validator("name", "John")
	Traceback (most recent call last):
	...
	TypeError: name must be an integer

CHECKING STRING INTEGER
        >>> bg.integer_validator("name", 2)

CHECKING INT AND STRING
	>>> bg.integer_validator(2, 2)

CHECKING AREA
	>>> bg.area()
	Traceback (most recent call last):
	...
	Exception: area() is not implemented

CHECKING BOOLEAN
	>>> bg.integer_validator(True, 2)

CHECKING BOOLEAN
	>>> bg.integer_validator("name", True)
	Traceback (most recent call last):
	...
	TypeError: name must be an integer

CHECKING INTEGER == 0
	>>> bg.integer_validator("name", 0)
	Traceback (most recent call last):
	...
	ValueError: name must be greater than 0

CHECKING INTEGER < 0
	>>> bg.integer_validator("name", 0)
        Traceback (most recent call last):
        ...
        ValueError: name must be greater than 0

CHECKING EMPTY 
	>>> bg.integer_validator()
	Traceback (most recent call last):
	...
	TypeError: integer_validator() missing 2 required positional arguments: 'name' and 'value'

CHECKING None
	>>> bg.integer_validator(None, None)
	Traceback (most recent call last):
	...
	TypeError: None must be an integer

CHECKING A STRING
	>>> bg.integer_validator("one")
	Traceback (most recent call last):
	...
	TypeError: integer_validator() missing 1 required positional argument: 'value'

CHECKING NO INT
	>>> bg.integer_validator("test", (5, 7))
	Traceback (most recent call last):
	...
	TypeError: test must be an integer

CHECKINH EMPTY STRING
	>>> bg.integer_validator("", 1)

CHECKING AREA FAIL
	>>> bg.area(1)
	Traceback (most recent call last):
        ...
	TypeError: area() takes 1 positional argument but 2 were given

CHECKING DICT
	>>> bg.integer_validator("dict", {"key": "value"})
	Traceback (most recent call last):
        ...
        TypeError: dict must be an integer

CHECKING SET
	>>> bg.integer_validator("set", {"hello", "world"})
        Traceback (most recent call last):
        ...
        TypeError: set must be an integer

CHECKING MANY ARGMENTS
	>>> bg.integer_validator("more", 1, 2)
	Traceback (most recent call last):
        ...
	TypeError: integer_validator() takes 3 positional arguments but 4 were given

CHECKING LIST
	>>> bg.integer_validator("list", [1, 2])
	Traceback (most recent call last):
        ...
        TypeError: list must be an integer
